src/containers/paint-editor.jsx

```diff
@@ -135,6 +141,10 @@ class PaintEditor extends React.Component {
         this.canvas = canvas;
     }
     onMouseDown () {
+        if (document.activeElement instanceof HTMLInputElement) {
```

Comment:
To keep this contained to this specific issue, would you also check that the mousedown event is on the paint editor canvas?

Reply:
Oops, thanks for catching